test/servico/cst20Test.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package servico;
+
+import auxiliar.Perfil;
+import datamapper.PopulateDB;
+import datamapper.exceptions.NonexistentEntityException;
+import dominio.Usuario;
+import java.util.List;
+import modelo.UsuarioModel;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Victor Messina
+ */
+public class cst20Test {
+
+    public cst20Test() {

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://www.codacy.com/app/calebepb/substituicao/file/2727989107/issues/source?bid=3308597&fileBranchId=3349084#l27)